src/components/DateTemperature/index.js

```diff
@@ -0,0 +1,56 @@
+import "macro-css";
+import { monthNames } from "../../utilities/mockData"
+import { MainBlockDay } from "./index1";
+
+
+
+export const MainBlock = ({ data, date }) => {
+	const temp = (n) => {
```

Comment:
all functions should name with an action. getTemperature, handleSomeone etc. what does mean n argument???? use arrow returning const funcName = (argName) => argName > 0 ? ... : ...